admin-dev/themes/new-theme/js/pages/currency/form/currency-form.js

```diff
@@ -138,6 +139,23 @@ export default class CurrencyForm {
     }
   }
 
+  _showResetDefaultSettingsConfirmModal() {
+    const confirmTitle = this.translations['confirmTitle'];
+    const confirmMessage = this.translations['confirmMessage'];
+    const confirmButtonLabel = this.translations['confirmButtonLabel'];
+    const closeButtonLabel = this.translations['closeButtonLabel'];
+
+    const modal = new ConfirmModal({
```

Comment:
@NeOMakinG looks like the modal you built is nicely reused 👍

Reply:
This part being in jquery however it makes total sens to use this component here :)